app/scripts/modules/tencentcloud/src/pipeline/stages/disableCluster/DisableCluster.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
```

Comment:
Please organize imports according to Deck's [import order conventions](https://github.com/spinnaker/deck/wiki/Conventions#imports). (Same comment for all other files.)